src/sensors/radiancemeter.cpp

```diff
@@ -0,0 +1,149 @@
+            // to_world, they must still be marked as queried
+            props.mark_queried("direction");
+            props.mark_queried("origin");
+        } else {
+            if (props.has_property("direction") !=
+                props.has_property("origin")) {
+                Throw("If the sensor is specified through origin and direction "
+                      "both values must be set!");
+            }
+
+            if (props.has_property("direction")) {
+                ScalarPoint3f origin     = props.point3f("origin");
+                ScalarVector3f direction = props.vector3f("direction");
+                ScalarPoint3f target     = origin + direction;
+                auto [up, unused]        = coordinate_system(direction);
+
+                m_world_transform = new AnimatedTransform(
+                    ScalarTransform4f::look_at(origin, target, up));
+            }
+        }
+
+        if (m_film->size() != ScalarPoint2i(1, 1))
+            Throw("This sensor only supports films of size 1x1 Pixels!");
+    }
+
+    std::pair<Ray3f, Spectrum> sample_ray(Float time, Float wavelength_sample,
+                                          const Point2f & /*position_sample*/,
+                                          const Point2f & /*aperture_sample*/,
+                                          Mask active) const override {
+        MTS_MASKED_FUNCTION(ProfilerPhase::EndpointSampleRay, active);
+        Ray3f ray;
+        ray.time = time;
+
+        // 1. Sample spectrum
+        auto [wavelengths, wav_weight] =
+            sample_wavelength<Float, Spectrum>(wavelength_sample);
+        ray.wavelengths = wavelengths;
+
+        // 2. Set ray origin and direction
+        auto trafo = m_world_transform->eval(time, active);
+        ray.o      = trafo.transform_affine(Point3f{ 0.f, 0.f, 0.f });
+        ray.d      = trafo.transform_affine(Vector3f{ 0.f, 0.f, 1.f });
+
+        return std::make_pair(ray, wav_weight);
+    }
+
+    std::pair<RayDifferential3f, Spectrum>
+    sample_ray_differential(Float time, Float wavelength_sample,
+                            const Point2f & /*position_sample*/,
+                            const Point2f & /*aperture_sample*/,
+                            Mask active) const override {
+        MTS_MASKED_FUNCTION(ProfilerPhase::EndpointSampleRay, active);
+        RayDifferential3f ray;
+        ray.time = time;
+
+        // 1. Sample spectrum
+        auto [wavelengths, wav_weight] =
+            sample_wavelength<Float, Spectrum>(wavelength_sample);
+        ray.wavelengths = wavelengths;
+
+        // 2. Set ray origin and direction
+        auto trafo = m_world_transform->eval(time, active);
+        ray.o      = trafo.transform_affine(Point3f{ 0.f, 0.f, 0.f });
+        ray.d      = trafo.transform_affine(Vector3f{ 0.f, 0.f, 1.f });
+
+        // 3. Set differentials
+        ray.d_x               = 0.f;
+        ray.d_y               = 0.f;
+        ray.has_differentials = true;
```

Comment:
Not sure whether we should even set the derivatives here. 
As this sensor operates on a single pixel it makes sense to that `d_x, d_y` should be equal to zero.

Maybe we could simply leave those with their default values (`ray.has_differentials` will be `false`)?